src/endpoints/proposals-inbox/domain/proposals-inbox.controller.ts

```diff
@@ -0,0 +1,52 @@
+import { Result } from '@alien-worlds/api-core';
+import { inject, injectable } from 'inversify';
+
+import {
+	ProposalsInboxOutput,
+	ProposalsInboxRequestDto,
+} from '../data/dtos/proposals-inbox.dto';
+import { ListProposalsUseCase } from './use-cases/list-proposals.use-case';
+
+/*imports*/
+
+/**
+ * @class
+ *
+ *
+ */
+@injectable()
+export class ProposalsInboxController {
+	public static Token = 'PROPOSALS_INBOX_CONTROLLER';
+
+	constructor(
+		@inject(ListProposalsUseCase.Token)
+		private listProposalsUseCase: ListProposalsUseCase /*injections*/
+	) {}
+
+	/*methods*/
+
+	/**
+	 *
+	 * @returns {Promise<Result<any>>}
+	 */
+	public async proposalsInbox(
+		input: ProposalsInboxRequestDto
```

Comment:
input should be `ProposalsInboxInput` which you created in the route pre-hook, then you dont have to do any parsing liek `limit: Number(input.limit)` because it should be done while creating this input model + please dont use camel_case in models only in dtos